### PR TITLE
update envoy-wasm

### PR DESCRIPTION
**Pull request by yxue**
Latest commit: https://github.com/envoyproxy/envoy-wasm/commit/ec03328688895d99c9f5ae4fd7f8459ef3e95212

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -35,9 +35,9 @@ bind(
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # envoy-wasm commit date: 09/24/2019
```
Comment (bianpengyuan) [inline WORKSPACE]: Update the date?

Comment (yxue) [reply WORKSPACE]: Good catch! Updated.

**Comment by PiotrSikora** [comment]
Can we wait until https://github.com/envoyproxy/envoy-wasm/pull/229 is merged? Otherwise we're going to update this and istio/istio twice.

cc @jplevyak

**Comment by yxue** [comment]
@PiotrSikora @bianpengyuan could you help to take a look at the PR?

**Pull request merged**